eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -86,6 +86,7 @@ steps:
     - template: /eng/pipelines/templates/send-to-helix.yml
       parameters:
         HelixProjectPath: '$(Build.SourcesDirectory)/tests/helix/send-to-helix-ci.proj'
+        # Code coverage on windows is disabled: https://github.com/dotnet/aspire/issues/6189
```

Comment:
Is it?

Reply:
Thanks, not anymore apparently. I apparently was looking at an older commit when I updated the comment.